storage/pruning/pruningStorer.go

```diff
@@ -242,7 +248,13 @@ func initPersistersInEpoch(
 		}
 	}
 
-	return persisters, persistersMapByEpoch, nil
+	ps.activePersisters = persisters
+	ps.persistersMapByEpoch = persistersMapByEpoch
+	return nil
+}
+
+func (ps *PruningStorer) shouldCloseOldPersisters(_ *persisterData) bool {
```

Comment:
why do we need this function for? It does not properly overridden by `triePruningStorer.shouldCloseOldPersisters`

Reply:
Refactored.